blog/_posts/2016-10-21-Team-Echo-wrapping-up.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Our end of RGSoC,:( !
+layout: post
+created_at: Thu July 28 2016
+permalink: blog/Team-Echo-wrapping-up-2016
+current: blog
+author: Naggita Keziah and Nanjekye Joannah
+twitter: _TeamEchoUg_
+categories:
+- student-posts
+---
+
+### Team Echo: Our RGSoC Landing
+
+Like any flight its like our RGSoC plane just landed .It has  been three awesome  months of learning, connecting, coding and changing lives. We pushed, pulled, squashed, force pushed, gave talks, revived Rails Girls Kampala, interviewed, toured, travelled and became official contributors to qutebrowser.
+
+The unforgettable memories of  updating teams app, responding to emails and slack messages, having our PRs merged, attending conferences, renewing Rails Girls Kampala and so much more made Rails Girls Summer of Code an amazing  experience. The summer is over and we are back to putting what we learnt and our experience to work in our communities.
+
+![team picture](/img/blog/2016/team_echo2.jpg)
```

Comment:
all images need image credits (who took the picture? Where is the image from?), you can find more about this [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images).